smassh/src/parser/parser.py

```diff
@@ -58,7 +55,7 @@ def _create_user_config(self) -> None:
         Creates a new config
         """
 
-        os.makedirs(self.config_path, exist_ok=True)
+        Path(self.config_path).makedir(parents=True, exist_ok=True)
```

Comment:
The `config_path` is already a `Path` object so no need to wrap it inside `Path` :)

Reply:
Done.